x/cronos/keeper/keeper.go

```diff
@@ -29,12 +29,12 @@ type (
 
 		// module specific parameter space that can be configured through governance
 		paramSpace paramtypes.Subspace
-		// evm parameter space
-		evmParamSpace paramtypes.Subspace
 		// update balance and accounting operations with coins
 		bankKeeper types.BankKeeper
 		// ibc transfer operations
 		transferKeeper types.TransferKeeper
+		// ethermint evm keeper
+		evmKeeper *evmkeeper.Keeper
```

Comment:
would be more clean to create a type.EvmKeeper

Reply:
Yeah, ideally that should be the case, the only obstacle currently is in [this method](https://github.com/crypto-org-chain/cronos/pull/34/files#diff-1cf88287bb640036f7fdeabdc381fba5bd0dc2dc5625d94846fca9359f1f2990R19), it calls many arbitrary methods of `EvmKeeper`, I'd like to finish our part first before pushing for a cleaner interface at upstream, then we change it into an interface.

Reply:
https://github.com/tharsis/ethermint/issues/533
Opened an issue in ethermint side.